### PR TITLE
Trying to get embed to work

### PR DESCRIPTION
**Pull request by ldecicco-USGS**

### CODE REVIEW

File: viz.yaml [hunk 1]
```diff
@@ -259,19 +259,41 @@ publish:
...
+    embedLink: embed-dots.html
     depends: ["lib-social-media-css"]
   -
     id: vertical-social-section
     template: vertical-social-media-template
     publisher: section
+    mainEmbed: true
+    facebookLink: http://www.facebook.com/sharer.php?u=https%3A%2F%2Fowi.usgs.gov%2Fvizlab%2FEXAMPLE%2F
+    twitterLink: https://twitter.com/share?url=https%3A%2F%2Fowi.usgs.gov%2Fvizlab%2FEXAMPLE%2F&text=EAMPLE1%20stuff&hashtags=EAMPLE1,EAMPLE2,dataviz
+    githubLink: https://github.com/USGS-VIZLAB/example
```
Comment (aappling-usgs) [inline viz.yaml]: just noticing that these sm links get replicated - is there a way to do this that only requires entering the links once, or are you pretty comfortable with this as the best solution?

Comment (ldecicco-USGS) [reply viz.yaml]: sm?

Comment (ldecicco-USGS) [reply viz.yaml]: I haven't really figured out how the regular and vertical social things work, with the example, it seems like you need both....but I still need to get it working, so, no, I'm not comfortable with this as the best solution

Comment (ldecicco-USGS) [reply viz.yaml]: like, in the RStudio preview, it's showing up horizontal, when I open it in Chrome, it's vertical, so both are being used, but via magic (probably, css...but that's less exciting)

Comment (aappling-usgs) [reply viz.yaml]: css and magic do have a lot in common =). sm = social media. OK, no worries, this piece is not very familiar to me either, just trying to keep an eye out for additional places where we can make improvements. we can come back to this later after the embedding stuff is sorted out.

**Comment by ldecicco-USGS** [comment]
Working links in social now:

image


(picture doesn't show it...but they are correct

**Comment by aappling-usgs** [comment]
Oops, sorry

**Pull request merged**